examples/webgl_lines_fat.html

```diff
@@ -223,6 +226,54 @@
 
 			//
 
+			function onDocumentMouseMove( event ) {
+
+				if ( ! interactive ) return;
+
+				event.preventDefault();
+
+				var intersects = getIntersects( event.layerX, event.layerY );
+
+				if ( intersects.length > 0 ) {
+
+					if ( prevLinewidth === undefined ) {
+
+						prevLinewidth = matLine.linewidth
+
+					}
+
+					var res = intersects[ 0 ];
+
+					if ( res && res.object ) {
+
+						res.object.selectedObject.material.linewidth = prevLinewidth + prevLinewidth / 3;
```

Comment:
It should be:
```js
res.object.material.linewidth
```
Otherwise you produce a runtime error: `Uncaught TypeError: Cannot read property 'material' of undefined`

Reply:
\>.< Sorry.. Fixed in 0a584ae45f